### DISCUSSION TITLE
Want to tur on LED with the press of any button

### DISCUSSION THREAD

**Discussion by sincos5** [tttapa/Control-Surface#538] [Q&A]
Hi! 
I have this code>
```cpp
#include <Control_Surface.h> // Include the Control Surface library

// Instantiate a MIDI over USB interface.
USBMIDI_Interface midi;

CCButtonLatched buttons [] = {
  {2, {MIDI_CC::General_Purpose_Controller_1,CHANNEL_2}},
  {3, {MIDI_CC::General_Purpose_Controller_2,CHANNEL_2}},
};

void setup() {
  Control_Surface.begin(); // Initialize Control Surface
}

void loop() {
  Control_Surface.loop(); // Update the Control Surface
}
```

And want to turn on a LED when any of the buttons is pressed.
I know i can make it via hardware, but the devise is already assembled and running, so it will be more difficult to do it on that way. 
Where can i find reference on this subject?
Thanks!

**Comment by tttapa** [comment]
Use the `CCButtonLatched::getState()` method.

You should be able to find some inspiration here: https://github.com/tttapa/Control-Surface/issues/241#issuecomment-660960589
